package.json

```diff
@@ -14,5 +15,8 @@
     "build:frontend": "pushd user-interface && npm run build && popd",
     "build:common": "pushd common && npm run build && popd",
     "clean-and-build": "npm run clean:all && npm run ci:all && npm run build"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "1.9.4"
```

Comment:
Biome 2.0 is [supposed to support plugins](https://github.com/biomejs/biome/discussions/1649). That said, I'm a bit concerned by the fact that as of yet, the only way to get new rules [is for the maintainers to implement the rules in Rust](https://github.com/biomejs/biome/discussions/1649#discussioncomment-11476815). To be clear, we don't often add rules. I've been wanting to add a rule (from a plugin or custom) to enforce newlines after functions/methods, but have been waiting until we handle the outdated package. We recently added `a11y` linting. So while we don't often add new rules, it does happen.